tests/ConfigurationTest.php

```diff
@@ -17,21 +17,16 @@
 use Lcobucci\JWT\Token\Parser as ParserImpl;
 use Lcobucci\JWT\Validation\Constraint;
 use Lcobucci\JWT\Validator;
+use PHPUnit\Framework\Attributes as PHPUnit;
 use PHPUnit\Framework\MockObject\MockObject;
 use PHPUnit\Framework\TestCase;
 
-/**
- * @covers ::__construct
- * @coversDefaultClass \Lcobucci\JWT\Configuration
- *
- * @uses \Lcobucci\JWT\Encoding\ChainedFormatter
- * @uses \Lcobucci\JWT\Encoding\MicrosecondBasedDateConversion
- * @uses \Lcobucci\JWT\Encoding\UnifyAudience
- * @uses \Lcobucci\JWT\Signer\Key\InMemory
- * @uses \Lcobucci\JWT\Token\Builder
- * @uses \Lcobucci\JWT\Token\Parser
- * @uses \Lcobucci\JWT\Validation\Validator
- */
+#[PHPUnit\CoversClass(Configuration::class)]
```

Comment:
I'd argue that summarizing all code execution inside this class is an incorrect generalization given that there are specific method names given that are to be covered.

Reply:
Given the attributes [only targets the class level](https://github.com/sebastianbergmann/phpunit/blob/main/src/Framework/Attributes/CoversMethod.php#L19), having a list of all the covered methods at the test class level would bring no benefits here.

Sebastian explains the reasoning behind his decisions here: https://github.com/sebastianbergmann/phpunit/issues/4502

Although I like the granularity, I understand the reasons and am fine with the changes in PHPUnit 10 and 11 :smile: